Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.h

```diff
@@ -150,7 +151,7 @@ typedef std::unordered_map<firebase::firestore::auth::User,
  *      simulating the server having sent a complete snapshot.
  */
 - (void)receiveWatchChange:(FSTWatchChange *)change
-           snapshotVersion:(FSTSnapshotVersion *_Nullable)snapshot;
+           snapshotVersion:(const firebase::firestore::model::SnapshotVersion &)snapshot;
```

Comment:
This was previously nullable and now is not.

Also, you've got a `#include` of `optional.h` but didn't use it.